pages.pt-BR/common/ab.md

```diff
@@ -0,0 +1,20 @@
+# ab
+
+> Ferramenta de benchmarking do servidor Apache. A ferramenta mais simples para realizar testes de carga.
+> Página oficial: <https://httpd.apache.org/docs/2.4/programs/ab.html>.
+
+- Executar 100 requisições HTTP do tipo GET para uma determinada URL:
```

Comment:
I'd suggest wrapping HTTP and GET in backticks.

Reply:
Why would that be necessary? Those are not other command names or special tokens. I've never seen backticks used in such cases in other pages.

See the following pages for example:

https://github.com/tldr-pages/tldr/blob/9c59940f45fd5371b190c2f170425d789a742cbb/pages/linux/mke2fs.md#L5

https://github.com/tldr-pages/tldr/blob/9c59940f45fd5371b190c2f170425d789a742cbb/pages/common/nasm.md#L5

https://github.com/tldr-pages/tldr/blob/9c59940f45fd5371b190c2f170425d789a742cbb/pages/common/less.md#L17

Reply:
@mebeim in my understanding, backticks serve to identify specific technical terms amidst regular English prose, such as commands, filenames, keyboard shortcuts, etc.

Granted, in this sentence HTTP could perhaps be considered a regular acronym, like URL, and be left unmarked; but GET is a specific HTTP verb that carries technical meaning in this context, so I still think it should be marked as code.

Reply:
I see. I would personally only use backticks for tokens and command names, but I get your point.

Reply:
I agree

Reply:
I'm not sure that HTTP warrants backticks as it's an abbreviation, right?

Reply:
@sbrl Acronym, but yeah, that was my point. Same for GET/POST etc, never saw those wrapped in backticks.

Reply:
So, do we have a consensus about that? For me, the protocol name and verbs should be without backticks?

Reply:
Well dunno, looks like a Mexican standoff :thinking: :rofl:

Reply:
My suggestion above, was:

> HTTP could perhaps be considered a regular acronym, like URL, and be left unmarked

so I'd only insist in "backticking" `GET` and `POST` (which are not acronyms, btw), again for the reason I suggested above:

> `GET` is a specific HTTP verb that carries technical meaning in this context, so I still think it should be marked as code.

Is that still a deal-breaker to you, @mebeim? From my reading of the discussion above it looks like this would be fine with @sbrl (but please confirm as well).

Reply:
@waldyrious I mean, for what I see that's not coherent with the rest of the pages. If we want to change GET into `GET` and POST into `POST` then we'd also have to change it elsewhere, not only in this page. Few examples:

https://github.com/tldr-pages/tldr/blob/dbf4e997968223ac4772c0634aa044cac0648685/pages/common/curl.md#L18

https://github.com/tldr-pages/tldr/blob/dbf4e997968223ac4772c0634aa044cac0648685/pages/common/mitmdump.md#L10

https://github.com/tldr-pages/tldr/blob/dbf4e997968223ac4772c0634aa044cac0648685/pages/linux/httpie.md#L5

https://github.com/tldr-pages/tldr/blob/dbf4e997968223ac4772c0634aa044cac0648685/pages/linux/httpie.md#L9

I also never saw GET or POST be treated specially inside code blocks in any `README.md`, Markdown documentation, blog posts, et cetera, so I don't really think that's necessary.

However, with the above being said, I would not argue if the majority of us thinks it's better to have those wrapped in backticks.

Reply:
I agree the change would need to happen in other pages as well to be consistent, but I don't think it's problematic to have this page kick-off that change and a later PR making the equivalent change to all other pages.

But if you guys either disagree with adding backticks to GET/POST etc. (I think both @andrik and @sbrl were OK with that, but please confirm), or think it's better for that change to happen entirely in a separate PR, that'd be fine by me.

Reply:
@waldyrious I think this could be done in a separate PR

Reply:
Unresolving this conversation to make it more discoverable. We had some trouble re-locating it in #3114 😅